west.yml

```diff
@@ -46,7 +46,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: fw-nrfconnect-zephyr
-      revision: e23d8ac48dd0e5890559d618c048132dd4b0a697
+      revision: 178775e5b117e18615d2a186bcb4dca8e56547ed
```

Comment:
It seems that the last commit has changed the revision. Is this one correct?
The PR in zephyr contained {{52417d3726f7888ed1a2c60b645ffb80a2fe7b56}}

Reply:
yea, talked with Carles.  going to wait for the upstream fix from zephyr.  
I manually stuck sphinx==2.4.4 in requirements-ci.txt, so we can keep CI running.